src/spec/sync/utils/product.spec.coffee

```diff
@@ -1151,6 +1151,46 @@ describe 'ProductUtils', ->
         ]
       expect(update).toEqual expected_update
 
+    it 'should not modify original attributes', ->
```

Comment:
In this particular case original product is used because in the new product, the old attribute is not present anymore and therefore needs to be removed from the original product.